### PR TITLE
minor language edits

### PR DESCRIPTION
**Pull request by JohnMarksRH**
Fixes issue # (delete if not relevant)

Changes proposed in this pull request:

- I made some very minor language edits.

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): @jmarks

This pull request needs review by: @mykaul

### CODE REVIEW

File: source/blog/newpost.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a test blog post. Getting to learn GitHub the real way.
```
Comment (rollandf) [inline source/blog/newpost.md]: Do you want this to be merged ?

Comment (JohnMarksRH) [inline source/blog/newpost.md]: @ rollandf. Correct, this ("This is a test post. Getting to learn..) is an unrelated file. I am new to Git and GitHub, hence I am making some errors along the way.

Comment (mykaul) [inline source/blog/newpost.md]: @JohnMarksRH - please remove the irrelevant file and push again.

**Comment by mykaul** [comment]
@JohnMarksRH - anything to do here, or should I close it?

File: source/blog/newpost.md (hunk 1 above)
Comment (JohnMarksRH) [inline source/blog/newpost.md]: @mykaul Please let me know if I resolved the issue.
